packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -90,12 +96,28 @@ export class ImportWallet extends React.Component<Props, State> {
   componentDidMount() {
     ValoraAnalytics.track(OnboardingEvents.wallet_import_start)
     this.props.navigation.addListener('focus', this.checkCleanBackupPhrase)
+
+    if (this.props.isRecoveringFromStoreWipe) {
+      this.autocompleteSavedMnemonic().catch((error) =>
+        Logger.error('Error while trying to recover account from store wipe:', error)
+      )
+    }
   }
 
   componentWillUnmount() {
     this.props.navigation.removeListener('focus', this.checkCleanBackupPhrase)
   }
 
+  async autocompleteSavedMnemonic() {
+    const wallet = await getWalletAsync()
+    const account = wallet?.getAccounts()[0]
```

Comment:
Though unlikely, there's a chance that this account value could differ from the one passed when initially opening `Screens.StoreWipeRecoveryScreen`, if we make some changes to this in the future.